src/dfmt/formatter.d

```diff
@@ -471,15 +474,18 @@ private:
         import dfmt.editorconfig : OptionalBoolean;
 
         immutable bool isCase = astInformation.caseEndLocations.canFindIndex(current.index);
-        immutable bool isAttribute = astInformation.attributeDeclarationLines.canFindIndex(current.line);
+        immutable bool isAttribute = astInformation.attributeDeclarationLines.canFindIndex(
```

Comment:
The algorithm is designed to prefer breaking lines at `(` over `.`. I don't think this is a bug.